.github/sign-images

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
For an example I don't think we need to include this, I assume that signed images it's something to configure later on, if someone wants really to (similarly, the default upgrade channel should be set with `verified: false`)